package/contents/ui/config/GoogleLoginManager.qml

```diff
@@ -72,49 +74,43 @@ Item {
 	signal sessionReset()
 	signal error(string err)
 
-
-	//---
 	readonly property string authorizationCodeUrl: {
 		var url = 'https://accounts.google.com/o/oauth2/v2/auth'
 		url += '?scope=' + encodeURIComponent('https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/tasks')
 		url += '&response_type=code'
-		url += '&redirect_uri=' + encodeURIComponent('urn:ietf:wg:oauth:2.0:oob')
+		url += '&redirect_uri=' + "http://127.0.0.1:" + callbackListenPort.toString() + "/"
```

Comment:
Does this not work when you wrap it in `encodeURIComponent()`? If it does I'll wrap it with that since there's slashes.

Reply:
Good point, I just added and tested with `encodeURIComponent` and it works fine.